lib/graphql/analysis/ast/query_complexity.rb

```diff
@@ -57,7 +57,8 @@ def on_leave_fragment_spread(node, _, visitor)
           visitor.leave_fragment_spread_inline(node)
         end
 
-        def result
+        # @return [Integer]
+        def max_possible_complexity
```

Comment:
This is the second, duplicate `#result`. I believe the renaming is needed, anyway.